### ISSUE TITLE
Importing after updating module creates duplicates

### ISSUE THREAD

**Issue by tbergstedt** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#143]
I have a FoundryVTT installation ongoing since 0.7.9, now "updated" to 0.8.8. What I had to do get 0.8.8 running was to copy the data to a completely new setup, separate from the old one and existing in parallel (the old one of course being shut down at the moment). 

Now when I update the game system and the FBL modules, I get the "Begin importing" dialogs at startup. Even though it says in the settings window that "previously existing entries will be overwritten", what actually happens is that I get duplicates of all items, characters, journal entries, etc. This is of course annoying, and I want to avoid it - but I still want to do the import in order to get any new macros, updates, etc. How can I do that?
